storage/src/lib.rs

```diff
@@ -81,3 +81,39 @@ pub use transaction::{abort, commit};
 
 pub type Data = Vec<u8>;
 pub type Result<T> = std::result::Result<T, Error>;
+
+#[cfg(test)]
+mod test {
+    use crate::{schema, traits::*};
+
+    struct MyColumn;
+    impl schema::Column for MyColumn {
+        const NAME: &'static str = "MyColumnV1";
+        type Kind = schema::Single;
+    }
+
+    type MySchema = (MyColumn, ());
+    type MyStore = crate::Store<MySchema>;
```

Comment:
Also, the reason why I don't like the word "Store" specifically, is that it can be both a noun and a verb depending on the context. I have always preferred "Database". "Storage" is also OK. Both are clear and unambiguous. I would feel happier if you chose one of these.

Reply:
I wanted to call it `Storage` but then the module should also be called `storage`, which is also a name of the crate. That caused some name resolution clashes in some contexts and was occasionally annoying to work with. I am fine with `Database` but would rather do the change in a subsequent PR, this one is messy enough already I reckon.